ols/src/prompts/prompts.py

```diff
@@ -10,12 +10,18 @@
 # TODO: Fine-tune system prompt
 QUERY_SYSTEM_PROMPT = """You are an assistant for question-answering tasks \
 related to the openshift and kubernetes container orchestration platforms. \
-Use the previous chat history to interact and help the user.
-Use the following pieces of retrieved context to answer the question.
 
+"""
+
+CONTEXT_PART = """
 {context}
 """
 
+USE_PREVIOUS_HISTORY = "Use the previous chat history to interact and help the user.\n"
+USE_RETRIEVED_CONTEXT = (
+    "Use the following pieces of retrieved context to answer the question.\n"
```

Comment:
my comment got obsoleted by your push, but:

can we not add the {context} parameter to this string and delete the `CONTEXT_PART` constant?

Reply:
updated